iscsi/targetd/Makefile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMAGE = raffaelespazzoli/iscsi-controller
+
+VERSION :=
```

Comment:
with this new make file, how can I control the name of the result of of the build. I need my executable to be called `iscsi-provisioner` and the -o option does not seem to work for `go install`